flap.py

```diff
@@ -71,6 +72,16 @@ def letterify(s): return "".join(LETTERS[c] for c in s)
     AD_TEXTS.append(ad)
 AD_STARTING_PADDING_SPACES = 70
 
+if int(''.join(map(lambda s: '%02d' % int(s), platform.mac_ver()[0].split('.')))) < 101501:
```

Comment:
`mac_ver()` just returns `('13.6', ('', '', ''), 'arm64')` for me, which means that this check actually returns *true* on my machine too - I'm guessing the version number has two dots in it for you?

I'm gonna push a small change that works on my machine and I *think* should continue to work for you.

Reply:
Alright well apparently i have no idea how to modify someone else's pull on GH hahahaha. I'll figure that out some other time, I'm on vacation. Here's some code that works for me and I think will still work for you:

```python
def mac_version_number():
    num = platform.mac_ver()[0]
    if num.count(".") < 2: num = num + ".0"
    num = "".join("%02d" % int(s) for s in num.split("."))
    return int(num)

# Some unicode is not supported on older macs :/
if mac_version_number() < 101501:
    # bummer, we have to use some old emoji that only kinda sorta look like what we wanted
    BLUE   = "🆒"
    GREEN  = "✅"
    WHITE  = "⬜"
    BROWN =  "🏾" # HACK: this is supposed to be a skin tone modifier
    YELLOW = "📒"
    ORANGE = "📙"
    RED    = X
```

Do you mind confirming that it works for you and pushing it if so? Happy to merge this after.

Reply:
Oh yeah, I guess I assumed it would be "13.6.0" there. Now I'm wondering if any platforms return simply "13".

Reply:
I pushed a change to keep the one-liner while being robust to one, two or three version number elements, see if it works.

Reply:
```
>>> int('{:<06s}'.format(''.join(map(lambda s: '{:02d}'.format(int(s)), platform.mac_ver()[0].split('.')))))
101306
>>> int('{:<06s}'.format(''.join(map(lambda s: '{:02d}'.format(int(s)), '13.6'.split('.')))))
130600
```

Reply:
Okay, so @antalsz pointed out that tuple comparison in Python is lexicographic, please hold.